src/map/status.cpp

```diff
@@ -12045,9 +12041,10 @@ static bool status_change_start_post_delay(block_list* src, block_list* bl, sc_t
 		case SC_HARMONIZE:
 			val2 = 5 + 5 * val1;
 			break;
+		case SC_WINKCHARM:
```

Comment:
Both could actually be moved up below SC_KILLING_AURA and removed here.

Reply:
Will check it.

Reply:
Done.